app/src/main/java/com/example/playlistmaker2/SearchActivity.kt

```diff
@@ -27,42 +30,49 @@ import retrofit2.Callback
 import retrofit2.Response
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
-
-
 class SearchActivity : AppCompatActivity() {
+
     private lateinit var binding: ActivitySearchBinding
 
     companion object {
         const val SEARCH_ACTIVITY = "SearchActivity"
         const val SEARCH_TEXT = "searchText"
         const val URL_API = "https://itunes.apple.com"
+        const val HISTORY_KEY = "key_for_history"
+        const val PREFS = "prefs"
     }
 
+    private val urlApi = "https://itunes.apple.com"
+
     private val retrofit = Retrofit.Builder()
+        .baseUrl(urlApi)
         .baseUrl(URL_API)
         .addConverterFactory(GsonConverterFactory.create())
         .build()
 
-    private val iTunseService = retrofit.create(ItunesService::class.java)
-
-    private val trackList = mutableListOf<Track>()
+    private val iTunesService = retrofit.create(ItunesService::class.java)
 
+    private val trackList = ArrayList<Track>()
 
+    private var recentTracks = ArrayList<Track>()
```

Comment:
```suggestion
    private val recentTracks = ArrayList<Track>()
```
Список и так мьютабельный, от мьютабельной переменной мало смысла